lib/Doctrine/Common/DataFixtures/ReferenceRepository.php

```diff
@@ -199,6 +199,8 @@ public function getReferenceNames($reference)
      * Checks if reference has identity stored
      *
      * @param string $name
+     *
+     * @return bool
```

Comment:
Any reasons for not using the native `: bool` return type?

Reply:
Yes: the class is not `final`, so that would be a BC-break.

Reply:
Good reason :smile: